core/templates/components/forms/custom-forms-directives/object-editor.directive.ts

```diff
@@ -36,6 +40,7 @@ angular.module('oppia').directive('objectEditor', [
         initArgs: '=',
         isEditable: '@',
         objType: '@',
+        schema: '=',
```

Comment:
getSchema: '&schema' would be better if you don't need the two-way binding here.

(Then you don't need getSchema below, I think. Make sure to test that it works.)